assets/fetch-normalized-vms.sql.j2

```diff
@@ -0,0 +1,44 @@
+#standardsql
+WITH
+
+###
+### Apply basic transformations to the Norway VMS positions.
+###
+raw_vms_positions_normalized AS (
+  SELECT
+    message_id as msgid,
+    `world-fishing-827.udfs_v20210701.normalize_shipname`(vessel_name) as shipname,
```

Comment:
You shouldn't be normalizing the ship name here, you want the name as it was received.

Reply:
Using vessel_name without normalization.